doc/usage/how-to-install.md

```diff
@@ -60,6 +61,20 @@
    2. The installer will prompt you to select a disk to target the installation. **Please do this carefully as the selected disk will be completely erased**.
    3. Wait for the installer to finish and disconnect the USB drive. If something fails during the installer, please retry.
 
+   **Note:** You can change the default root password of the current instalation by executing the installer like this
+   ```
+    ./install.sh --root-password my_password
+   ```
+   Keep in mind that if you want to change the password for every latter instalations that you might create, instead of manually passing the password each time in the installer, follow the following steps
```

Comment:
Hmm, is this actually possible? 
If I understod correctly `change-password.sh` it does depend on moving the new `05-password.hsl` to the RW iso directory as it depends on the `install.sh` copied files. But even if `install.sh` can work with the ISO mounted as Read-Only, it would fail if the password is passed as the change-password script would fail.

Reply:
Yep, it is now possible and I already tested it, althought it lacked a now applied patch in `change-password.sh`, so no matter where is it called from, it now uses the `/tmp/` folder as a lab directory, meaning it only receives the path where the system files are, copies the `05-password.hsl` to the `/tmp` folder, patches the file there and copies back the file.

In the case of changing the password from the ISO, the script receives the path where the usb is mounted, so it replaces the file directly in the usb, so as the checksum and the instalation script completes with an updated password successfully :D .

Reply:
Oh i get it. Wow it sound super cool. I'll test this as soon as I can!